ROMFS/px4fmu_common/init.d/rc.interface

```diff
@@ -138,7 +138,18 @@ then
 	then
 		if fmu mode_$AUX_MODE
 		then
-			if [ -e $OUTPUT_AUX_DEV ]
+			# Append aux mixer to main device
+			if [ $OUTPUT_MODE == hil ]
+			then
+				if mixer append $OUTPUT_DEV $MIXER_AUX_FILE
+				then
+					echo "INFO  [init] Mixer: $MIXER_AUX_FILE appended to $OUTPUT_DEV"
+				else
+					echo "ERROR [init] Error appending mixer: $MIXER_AUX_FILE"
+					echo "ERROR [init] Could not append mixer: $MIXER_AUX_FILE" >> $LOG_FILE
+				fi
+			fi
+			if [ -e $OUTPUT_AUX_DEV -a $OUTPUT_MODE != hil ]
```

Comment:
Yes I'm running this on a Pixhawk, so when I use the airframe without HIL, the console shows:
```
INFO  [init] Mixer: /etc/mixers/octo_vtol.main.mix on /dev/pwm_output0 
INFO  [init] Mixer: /etc/mixers/vtol_tailsitter_AAEE.aux.mix on /dev/pwm_output1 
```

Then I added this at the bottom of the airframe:
`set HIL yes`

Which gives:
```
INFO  [init] Mixer: /etc/mixers/octo_vtol.main.mix on /dev/pwm_output0 
INFO  [init] Mixer: /etc/mixers/vtol_tailsitter_AAEE.aux.mix appended to /dev/pwm_output0 
```